<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_menue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>971</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QPushButton" name="optionen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>555</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>Optionen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="multiplayer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiplayer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zeitrennen">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>345</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>Zeitrennen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ueben">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>Üben</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lernen">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>135</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>Lernen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="anmelden">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string>Anmelden</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>325</y>
       <width>150</width>
       <height>6</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>115</y>
       <width>150</width>
       <height>6</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>535</y>
       <width>150</width>
       <height>6</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>-10</y>
       <width>20</width>
       <height>1191</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_lernen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>771</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_text">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>520</y>
       <width>750</width>
       <height>31</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>791</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>750</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_uebungende">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>570</y>
       <width>220</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Übung abbrechen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_starten">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>570</y>
       <width>220</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Übung starten</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_hand">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>611</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="gb_links">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>180</width>
       <height>400</height>
      </rect>
     </property>
     <property name="title">
      <string>Linke Hand</string>
     </property>
     <widget class="QPushButton" name="bl_zeige">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>160</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>Zeigefinger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bl_kleiner">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>160</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>Kleiner Finger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bl_mittel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>160</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>Mittelfinger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bl_ring">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>160</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>Ringfinger</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gb_rechts">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>180</width>
       <height>400</height>
      </rect>
     </property>
     <property name="title">
      <string>Rechte Hand</string>
     </property>
     <widget class="QPushButton" name="br_zeige">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>160</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>Zeigefinger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="br_kleiner">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>160</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>Kleiner Finger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="br_mittel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>160</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>Mittelfinger</string>
      </property>
     </widget>
     <widget class="QPushButton" name="br_ring">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>160</width>
        <height>75</height>
       </rect>
      </property>
      <property name="text">
       <string>Ringfinger</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
